### DISCUSSION TITLE
👋 oh, um, hello there!

### DISCUSSION THREAD

**Discussion by tacosontitan** [tacosontitan/tacosontitan#2] [Announcements]
## 👋 Welcome!
Since GitHub doesn't allow direct messages, and I'm not on Twitter enough to respond actively there, feel free to use the discussions area to reach out and communicate with me!

**Comment by RutulPatel8** [comment]
Hi @tacosontitan
Rutul here from india.

I just checked your profile. It's just amazing. Can you please help me create this type of profile?

**Reply by tacosontitan** [reply]
👋 have you reviewed my personal repository's [`README.md` file](https://raw.githubusercontent.com/tacosontitan/tacosontitan/main/README.md)? I'd be more than happy to assist you with any questions you have about modifying it.

**Reply by RutulPatel8** [reply]
Yes I read it. Its incredible. Can you please what should I add in my profile to make it attractive like you.

**Reply by tacosontitan** [reply]
I mean, I can't create it for you, and without you giving some ideas on what you want to sell yourself with, I can't really advise for or against anything other than what's already in my current README, which was purely my personal preference.

Still happy to help, I'm just not sure how without you providing some details.

**Reply by Programmer-Faraj** [reply]
Hi there Hazel,

It's a big pleasure for me to find your profile and connect with you here in GitHub.

**Reply by Programmer-Faraj** [reply]
I'm just like you. I love coding and this wondaful technology feels just like playing games. I have a background in C# but I've been taking the CS50x course hosted by Harvard University and I am having sooooo much fun learning multiple languages.

**Reply by Programmer-Faraj** [reply]
Are you on Discord?

**Reply by tacosontitan** [reply]
Yes, I'm an active member of the C# server there.

**Reply by Programmer-Faraj** [reply]
I'm not a member in the C# server, but I would love to connect with you on Discord if you can paste your link? This is my profile.
![image](https://github.com/user-attachments/assets/61accb8a-a06d-4a6d-9a88-b6172249a331)
